webextensions/common/tabs-store.js

```diff
@@ -171,31 +171,38 @@ function* getMatchedTabsIterator(tabs, query, offset) {
 }
 
 function matchedWithQuery(tab, query) {
-  for (const attribute of MATCHING_ATTRIBUTES) {
+  for (let i = 0, maxi = MATCHING_ATTRIBUTES.length; i < maxi; i++) {
```

Comment:
Very interesting. I didn't thought that "for-of" loop is worse than simple "for" loop. To prevent mis-reverting by me this in the future, could you add a comment (the commit message itself looks suitable) here?